docs/reference/faq.md

```diff
@@ -21,6 +21,17 @@ to make relative file paths clear.
 Most Broker subcommands allow customizing the data root via the `-r` flag.
 For more information on this and other runtime customization, run `broker -h`.
 
+### Can I customize the temporary directory used by Broker?
+
+- On Linux and macOS: set the `$TMPDIR` environment variable.
```

Comment:
[nit] I know we're prefixing env vars with `$` in the rest of this document, but I think we should only do this when we're showing a variable in an expression, not when we are referring to the name of the variable.

Reply:
Good call, changed!